extensions/rocksdb-repos/FlowFileRepository.cpp

```diff
@@ -69,14 +68,8 @@ void FlowFileRepository::flush() {
     batch.Delete(keys[i]);
   }
 
-
-  if (db_->Write(rocksdb::WriteOptions(), &batch).ok()) {
-    logger_->log_trace("Decrementing %u from a repo size of %u", decrement_total, repo_size_.load());
-    if (decrement_total > repo_size_.load()) {
-      repo_size_ = 0;
-    } else {
-      repo_size_ -= decrement_total;
-    }
+  if (!db_->Write(rocksdb::WriteOptions(), &batch).ok()) {
+    logger_->log_warn("Failed to execute batch operation when flushing FlowFileRepository");
```

Comment:
I removed size handling as:
- It was done in a inconsistent way.
- Even if the repo_full flag was set, nothing happened, new elements were inserted. 
- Handling repo size out of the repo (without taking the size of keys into account) is most probably a bad design.

Reply:
Agreed, but I think it is something we would need, done properly. Is there a follow-up issues for it?

Reply:
https://issues.apache.org/jira/browse/MINIFICPP-1129